rules/windows/privilege_escalation_printspooler_suspicious_spl_file.toml

```
@@ -21,26 +21,31 @@ rule_id = "a7ccae7b-9d2c-44b2-a061-98e5946971fa"
 severity = "high"
 tags = ["Elastic", "Host", "Windows", "Threat Detection", "Privilege Escalation"]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.category:file and not event.type:deletion and
- file.extension:(spl or SPL) and
- file.path:C\:\\Windows\\System32\\spool\\PRINTERS\\* and
- not process.name:(spoolsv.exe or printfilterpipelinesvc.exe or PrintIsolationHost.exe or splwow64.exe or msiexec.exe or poqexec.exe)
+file where event.type != "deletion" and
+  file.extension : ("spl", "SPL") and
```

case-insensitive, this doesn't matter

```suggestion
  file.extension : "spl", "SPL" and
```

---

nice catch

```suggestion
  file.extension : "spl" and
```